packages/admin/src/pages/super-admin-dashboard/user/[id]/change-roles.page.tsx

```diff
@@ -21,31 +21,43 @@ type PageBodyResponse = {
   newUserRoles: string | string[];
 };
 
+const ROLE_IDS = {
+  FIND: '1',
+  APPLICANT: '2',
+  ADMIN: '3',
+  SUPER_ADMIN: '4',
+  TECH_SUPPORT: '5',
+};
+const APPLICANT_ROLES_IDS = [ROLE_IDS.FIND, ROLE_IDS.APPLICANT];
+const ADMIN_ROLES_IDS = [
+  ROLE_IDS.ADMIN,
+  ROLE_IDS.SUPER_ADMIN,
+  ROLE_IDS.TECH_SUPPORT,
+];
+
+function hasAdminRole(roles: string[]) {
+  return roles.some((role) => ADMIN_ROLES_IDS.includes(role));
+}
```

Comment:
if you're only ever feeding this an array of roles then you can just write the function to expect that

```suggestion
function hasAdminRole(roles: Role[]) {
  return roles.some(({ id }) => ADMIN_ROLES_IDS.includes(id));
}
```

then below you can just do

```js
const isAdmin = hasAdminRole(roles);
```

Reply:
in this case we're using this function elsewhere in the getServerSideProps - it was just extracted out of getServerSideProps to reduce duplicating logic, so changing the parameter type would mean mapping arrays of ids to `Role[]` in a few places since that's how the form posts. 

But would be good to rename the variables like
```suggestion
function hasAdminRole(roleIds: string[]) {
  return roleIds.some((roleId) => ADMIN_ROLES_IDS.includes(roleId));
}
```
to make it more clear what the function is doing

Reply:
I've actually implemented both now since we had a need for both in multiple places